### PR TITLE
Initial Product list UI for cell

### PR DESCRIPTION
**Pull request by syedtazaidi**

### CODE REVIEW

File: CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//  Created by Taha Ali on 2022-05-03.
+//
+
+import Foundation
+import UIKit
+
+/// A view that displays a star rating.
+public class StarsView: UIView {
+  /// The rating to display.
+  public var rating: Int = 0 {
```
Comment (designerGenes) [inline CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift:14]: this implementation code probably belongs in the presenter

**Review by designerGenes — APPROVED** [review]
needs a presenter.  we can handle that on another branch if you'd like though

File: CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift (hunk 1 above)
Comment (syedtazaidi) [reply CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift:14]: Isn't this purely related to StarsView to show images based on the rating?  This is the property we need to assign the actual rating which will be coming from the network. THis is the just initialization within the View.  Let me know if does not make sense

Comment (designerGenes) [reply CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift:14]: I thought I was still looking at the view controller.  This should be fine

**Review by designerGenes — APPROVED** [review]

File: CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift (hunk 1 above)
Comment (syedtazaidi) [reply CapitalOneShoppingCartDemo/CapitalOneShoppingCartDemo/ProductList/StarRatingsView.swift:14]: @designerGenes  @RituIosDev  @prash-mobiledev  I think we should merge this and all code for 2nd screen today before going forward . For now, this is the UI I'm getting. I will make it cleaner once we have all the Data Model setup once the Data persistence logic is merged. 

image

**Review by prash-mobiledev — APPROVED** [review]
Looks good

**Pull request merged**